test/NuGet.Clients.Tests/NuGet.VsExtension.Test/VsCredentialProviderImporterTests.cs

```diff
@@ -28,8 +28,9 @@ public class VisualStudioAccountProvider : IVsCredentialProvider
 
     public class VsCredentialProviderImporterTests
```

Comment:
Added two tests (one for dev14, one for dev15 behavior), verifying that a failing imported provider is logged and does not prevent other imported providers from loading (including the built in provider on dev14). See https://github.com/NuGet/NuGet.Client/pull/664/commits/a5a979f7192c3a68e4ca2f0f52ea07fa4d4f9435